quantumdb-query-rewriter/src/main/java/io/quantumdb/query/rewriter/PostgresqlQueryRewriter.java

```diff
@@ -51,31 +51,32 @@ public String rewrite(String query) throws SQLException {
 		for (state.cursor = 0; state.cursor < query.length(); state.cursor++) {
 			char currentCharacter = query.charAt(state.cursor);
 
-				switch (currentCharacter) {
-					case '\'':
-					case '\"':
-					case '`':
-						Character onStack = state.stack.peekFirst();
-						if (onStack != null && onStack.equals(currentCharacter)) {
-							state.stack.removeFirst();
-						}
-						else {
-							state.stack.addFirst(currentCharacter);
-						}
-						processWordStartOrMiddle(state);
-						break;
-					case ' ':
-					case ',':
-					case ';':
-						if (state.stack.isEmpty()) {
-							processWordEnding(state);
-						}
-						processWordStartOrMiddle(state);
-						break;
-					default:
-						processWordStartOrMiddle(state);
-						break;
-				}
+			switch (currentCharacter) {
+				case '\'':
+				case '\"':
+				case '`':
+					Character onStack = state.stack.peekFirst();
+					if (onStack != null && onStack.equals(currentCharacter)) {
+						state.stack.removeFirst();
+					}
+					else {
+						state.stack.addFirst(currentCharacter);
+					}
+					processWordStartOrMiddle(state);
+					break;
+				case ' ':
+				case '\t':
```

Comment:
Maybe \n (line breaks) should also be included, but I am not sure as I have not tested it.

Reply:
For now I'd suggest to leave it out then.